bigtop-packages/src/charm/hive/layer-hive/lib/charms/layer/bigtop_hive.py

```diff
@@ -69,35 +83,44 @@ def configure_hive(self):
         config = hookenv.config()
         hive_env = self.dist_config.path('hive_conf') / 'hive-env.sh'
         utils.re_edit_in_place(hive_env, {
-            r'.*export HADOOP_HEAPSIZE *=.*': 'export HADOOP_HEAPSIZE=%s' % config['heap'],
```

Comment:
Looks like the rest of the changes to this file are lint fixes.  While they are much appreciated, in the future, please make them in a separate PR.  It's a bit harder to focus on the functional code changes with these formatting changes.